components/centraldashboard-angular/frontend/scripts/update-environment-prod-ts.js

```diff
@@ -0,0 +1,10 @@
+const fs = require('fs');
+
+const args = process.argv.slice(2);
+const file = args[0];
+const build_version = args[1];
+
+let fileString = fs.readFileSync(file).toString();
+fileString = fileString.replace('${BUILD_VERSION}', build_version);
+
+fs.writeFileSync(file, fileString);
```

Comment:
Let's throw an error if `file` or `build_version` is undefined. Maybe we can also change the script's file name since it can be used in other files and not only for `environment.prod.ts`.